Makefile

```diff
@@ -103,6 +106,9 @@ signed_xpi: addon
 	./node_modules/.bin/web-ext sign --api-key=${AMO_USER} --api-secret=${AMO_SECRET} --source-dir addon/webextension/
 	mv web-ext-artifacts/*.xpi build/pageshot.xpi
 
+addon_locales: $(l10n_dest)
```

Comment:
I'm not confident exactly how this will work.  Generally the Makefile pattern would be to set the requirement and then have other rules for how to generate `addon/webextension/_locales/%/messages.json`, like:

```
addon/webextension/_locales/%/messages.json: locales/%/webextension.properties
  script-that-translates-one-file $< $@
```

Since this appears to generate all the files in one go, it doesn't quite fit that pattern.  I can't decide if $(l10n_source) belongs here, or maybe not, or maybe nothing at all?

Also there should be `.PHONY: addon_locales` above this